### ISSUE TITLE
Add mindmap / notes on other blockchains

### ISSUE THREAD

**Issue by truedat101** [IoTone/TheShellProject#20]
Sketch up some stuff on the other blockchains we are evaluating

**Comment by truedat101** [comment]
IOTA Tangle Mindmap

![20180510_224330570_ios](https://user-images.githubusercontent.com/64202/39898394-556ea060-546b-11e8-8193-c948b91404f0.jpg)

**Comment by truedat101** [comment]
@pedroalvesbatista can you add your mindmap for the bytecoin whitepaper?  Saves me a read.

**Comment by truedat101** [comment]
I'll write up NEM also (original).

**Comment by pedroalvesbatista** [comment]
This weekend I can add up my mindmap of NEM, EOS and Bytecoin as well. I think the Ripple transaction flow is worth to take into account for this mind process.

**Comment by truedat101** [comment]
Yep, please add an issue for each, and probably we put into the wiki.
